apps/webapp/app/routes/magic.tsx

```diff
@@ -7,6 +7,6 @@ export async function loader({ request }: LoaderArgs) {
 
   await authenticator.authenticate("email-link", request, {
     successRedirect: redirectTo ?? "/",
-    failureRedirect: "/login",
+    failureRedirect: "/login/magic",
```

Comment:
This seems to make sense. We should probably do this for the GitHub strategy as well.

@matt-aitken Can you see any issues? Was there a particular reason we'd want to always redirect to `/login` on errors instead of staying on the current strategy-specific page? (There's a back button on `/login/magic` for example.)

If we don't do this, user-facing errors won't actually be displayed until we navigate to `/login/magic` again.